### ISSUE TITLE
Adding collection in RDM fails with TypeError

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#9884]
1. Open RDM
2. Add a collection

```py
2023-08-08 09:45:50 Internal Server Error: /concepts/0ba52e5d-b5ff-48f5-b494-cc431e37aaf4
2023-08-08 09:45:50 Traceback (most recent call last):
2023-08-08 09:45:50   File "/root/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 47, in inner
2023-08-08 09:45:50     response = get_response(request)
2023-08-08 09:45:50   File "/root/.local/lib/python3.10/site-packages/django/utils/deprecation.py", line 119, in __call__
2023-08-08 09:45:50     response = self.process_response(request, response)
2023-08-08 09:45:50   File "/root/.local/lib/python3.10/site-packages/django/middleware/clickjacking.py", line 26, in process_response
2023-08-08 09:45:50     if response.get('X-Frame-Options') is not None:
2023-08-08 09:45:50 TypeError: HttpResponseBase.get() missing 1 required positional argument: 'header'
2023-08-08 09:45:50 [08/Aug/2023 13:45:50] "GET /concepts/0ba52e5d-b5ff-48f5-b494-cc431e37aaf4?f=html HTTP/1.1" 500 97080
```

### Version ###
7.4

Will submit a PR with the fix.

**Comment by jacobtylerwalls** [comment]
It's possible you many need an invalid concept name to trigger this, or perhaps this is downstream of the issue id'd at #9870, but either way it's worth addressing this independent issue.

**Comment by jacobtylerwalls** [comment]
Fixed in #9885
